test/groovy/com/sap/piper/integration/TransportManagementServiceTest.groovy

```diff
@@ -100,9 +103,12 @@ class TransportManagementServiceTest extends BasePiperTest {
 
         shellRule.setReturnValue(JenkinsShellCallRule.Type.REGEX, ".* curl .*", {throw new AbortException()})
 
+        readFileRule.files << [ 'responseFileUpload.txt': 'Something went wrong during file upload']
+
         thrown.expect(AbortException.class)
+        loggingRule.expect('Something went wrong during file upload')
 
-        def tms = new TransportManagementService(nullScript, [:])
+        def tms = new TransportManagementService(nullScript, [verbose:true])
```

Comment:
Could you also please provide a test for not verbose case of failing file upload?

Reply:
yes

Reply:
done